docs/usage/initialization.rst

```diff
@@ -101,3 +79,60 @@ Python client initialization:
 
 
 For more information on client side SSL cerificationcas, please refer to this [gist](https://gist.github.com/mtigas/952344).
+
+Dealing with errors during parsing metadata
+-------------------------------------------
+
+Class config provides easy to use wrapper for all parser configuration. These are:
+    - XML namespaces
+    - Parser policies(how parser act in case of invalid XML tag). We now support three types of policies:
+        - Policy fatal - parser raises exception and terminates
+        - Policy warning - parser reports given problem, then execute fallback code(e. g. replace unknown type with NullType) and then continues normally.
```

Comment:
`Policy warning - the policy reports given problem, then executes a fallback code(e. g. replace unknown type with NullType) and then continues normally